docs/zkapps/secure-zkapps.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Security and zkApps
+hide_title: true
+description: An introduction to security considerations for zkApps, including common pitfalls and best practices.
+keywords:
+  - zkApp
+  - smart contract
+  - typescript
+  - o1js
+  - mina protocol
+  - zero knowledge proofs
+---
+
+:::info
+
+zkApp programmability is not yet available on the Mina Mainnet, but zkApps can now be deployed on Berkeley Testnet.
+
+:::
```

Comment:
```suggestion
:::info

zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps can now be deployed on the Mina **Devnet**.

:::
```

Reply:
shouldn't we just change this for all pages at once?